docker/cron/backup.sh

```diff
@@ -12,11 +12,11 @@ set -e
 ####
 
 # backup everything to private archive
-echo "Extracting full backup..."
-pg_dump -Fc openstatesorg > openstatesorg.pgdump
-echo "Shipping full backup to s3"
-aws s3 cp openstatesorg.pgdump "s3://openstates-backups/full-backup/$(date +%Y-%m-%d)-openstatesorg.pgdump" > /dev/null
-rm -f openstatesorg.pgdump
+#echo "Extracting full backup..."
```

Comment:
Can you please add a comment line explaining why this is commented out? in case someone stumbles on it on it in the future...